compiler/lib/freevars.mli

```diff
@@ -25,4 +25,10 @@ val iter_block_bound_vars : (Code.Var.t -> unit) -> Code.block -> unit
 
 val f_mutable : Code.program -> Code.Var.Set.t Code.Addr.Map.t
 
+val iter_instr_free_vars : (Code.Var.t -> unit) -> Code.instr -> unit
```

Comment:
can you move the `iter_*` functions next the ones above

Reply:
Or rather, move `f_mutable` next to `f`

Reply:
Done